specification/playwrighttesting/resource-manager/readme.md

```diff
@@ -29,7 +29,26 @@ title: PlaywrightTestingClient
 description: Azure Playwright testing management service
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2023-10-01-preview
+tag: package-2024-02-01
+```
+
+### Tag: package-2024-02-01
+
+These settings apply only when `--tag=package-2024-02-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-02-01'
+input-file:
+  - Microsoft.AzurePlaywrightService/stable/2024-02-01/playwrighttesting.json
+suppressions:
+  - code: ResourceNameRestriction
+    from: playwrighttesting.json
+    reason: We have enums defined for quota resource names which doesn't support string pattern validation.
+  - code: ProvisioningStateMustBeReadOnly
+    from: playwrighttesting.json
+    reason: Seems like a tool bug, as the visibility is set to read only in the TypeSpec already.
+  - code: DeleteResponseCodes
+    from: playwrighttesting.json
```

Comment:
Please provide full path where this suppression is being applied. Whole file name is not correct.

Reply:
Done